src/esthri/src/main.rs

```diff
@@ -44,6 +44,9 @@ struct Cli {
 enum Command {
     /// Upload an object to S3
     Put {
+        /// Should the file be compressed during upload
+        #[structopt(long)]
+        compress: bool,
         #[structopt(long)]
         bucket: String,
```

Comment:
If the comments here are what make it to the cli usage/help maybe it would be worth adding the format the s3 bucket should be in? e.g.
"drive-logs2/path/to/log"
or
"s3://drive-logs2/path/to/log"

Reply:
there's no special format for these parameters, but updated the CLI docs to reflect that